### ISSUE TITLE
verifying module: checksum mismatch

### ISSUE THREAD

**Issue by KaptinLin** [uber-go/fx#1047]
``` go get go.uber.org/fx@v1
go: downloading go.uber.org/dig v1.16.0
go.uber.org/fx imports
        go.uber.org/dig: go.uber.org/dig@v1.16.0: verifying module: checksum mismatch
        downloaded: h1:UvbC1KUaQKx6MQTALcKanqRuPQRX7Tnt1iIYZHH2shk=
        sum.golang.org: h1:O48QoUEj4ePocypAIE5jz+SrxVdG/izHM1CZ/Yjrwww=```

**Comment by abhinav** [comment]
@sywhang @r-hang consider tagging a patch release with #1029 cherry picked into it so that "latest version of Fx" does not attempt to download the broken dig.

**Comment by sywhang** [comment]
[v1.19.2](https://github.com/uber-go/fx/releases/tag/v1.19.2) has been released with Dig v1.16.1. Sorry for the inconvenience.
